### ISSUE TITLE
Installing DaisyUI causes 'neutral' Tailwind class to be unavailable

### ISSUE THREAD

**Issue by Petercopter** [saadeghi/daisyui#683]
After installing DaisyUI, the Tailwind classes like 'bg-neutral-400' are no longer available. The only one that works is now the 'neutral' class coming from DaisyUI. Is there a way to allow those classes to work alongside DaisyUI, or am I doing something wrong?

It's weird because I'm searching the repo for the word 'neutral' and I'm not seeing all the search results I should be...

**Comment by saadeghi** [comment]
The problem is, when I named the colors for daisyUI, Tailwind CSS didn't have a `neutral` color name. It had `gray`, `warmGray`, `coolGray`, etc...  
And when they added a `neutral` color name, I couldn't just rename the color because:
1. It would break existing apps.
2. there are no better words for it. `neutral` is the best word to describe this color on daisyUI because it's neutral about the brand colors (primary, secondary, etc.) dark/light mode, and states(success, warning, etc.)

So, it remained as `neutral`.  
One way is disabling daisyUI `neutral` (if you never want to use daisyUI neutral colors like `bg-neutral` or `text-neutral-content`)  
https://play.tailwindcss.com/XP44zIozPw?file=config
```js
const colors = require('tailwindcss/colors')

module.exports = {
  plugins: [require('daisyui')],
  theme: {
    extend: {
      colors: {
        neutral: colors.neutral,
      },
    },
  },
}

```
Another way is to use a different name to have all colors together:  
https://play.tailwindcss.com/nuB4LP9iLl?file=config
```js
const colors = require('tailwindcss/colors')

module.exports = {
  plugins: [require('daisyui')],
  theme: {
    extend: {
      colors: {
        neutralgray: colors.neutral,
      },
    },
  },
}

```
```html
<div class="p-10 bg-neutralgray-400"></div>
```

**Comment by ekwoka** [comment]
I just want to voice that I disagree with the logic to not adjust daisy to be compatible with existing tailwind apps.

Breaking new adopters apps is a bigger issue than having a breaking change for users that can stay on a prior version.

If it's going to be based in tailwindcss, it needs to be compatible with tailwindcss default setups.

**Comment by ylor** [comment]
Was also bitten by this.

I would also agree that daisyUI "should" (rather than needs to) follow Tailwind defaults and enhance/extend.

**Comment by saadeghi** [comment]
Fixed in latest version `2.49.0`
